test/integration/start_stop_delete_test.go

```diff
@@ -166,6 +166,46 @@ func TestStartStop(t *testing.T) {
 	})
 }
 
+func TestStartStopWithPreload(t *testing.T) {
+	if NoneDriver() {
+		t.Skipf("skipping %s - incompatible with none driver", t.Name())
+	}
+
+	profile := UniqueProfileName("test-preload")
+	ctx, cancel := context.WithTimeout(context.Background(), Minutes(40))
+	defer CleanupWithLogs(t, profile, cancel)
+
+	startArgs := []string{"start", "-p", profile, "--memory=2200", "--alsologtostderr", "-v=3", "--wait=true"}
```

Comment:
Can you modify this test so that it first starts with an older release of Kubernetes (v1.14.0) that is not preloaded?

That will help the test distinguish between which behavior is fixed, because otherwise the test may be passing simply because it knows that preload isn't necessary because it has already been done.

Reply:
That's a good point, done.